osquery/tables/system/darwin/homebrew_packages.cpp

```diff
@@ -56,10 +54,35 @@ std::vector<std::string> getHomebrewVersionsFromInfoPlistPath(
   return results;
 }
 
+fs::path getHomebrewPrefix() {
+  // The Homebrew wrapper script finds the Library directory by taking the
+  // directory that it is located in and concatenating `/../Library`:
+  //   BREW_FILE_DIRECTORY=$(chdir "${0%/*}" && pwd -P)
+  //   export HOMEBREW_BREW_FILE="$BREW_FILE_DIRECTORY/${0##*/}"
+  // Note that the `-P` flag to pwd resolves all symlinks.
+  //
+  // Next, it will use given filename to find the prefix:
+  //   HOMEBREW_PREFIX = Pathname.new(HOMEBREW_BREW_FILE).dirname.parent
+
+  // Get the actual location of the Homebrew binary.
+  // In the future, we could extend this to look at all 'brew' executables in
+  // $PATH and check all of them.
+  auto brewExecutable = fs::canonical("/usr/local/bin/brew");
```

Comment:
Above the `kHomebrewBinary` symbol is used:
```cpp
kHomebrewBinary = "/usr/local/bin/brew";
```

Then below the static string is repeated. So it *is* checked for existence, but we're repeating const strings, think of the wasted bytes!!!! Oh the humanity!!!!!